packages/babel-plugin-transform-classes/src/vanilla.js

```diff
@@ -522,9 +522,18 @@ export default class ClassTransformer {
       if (this._processMethod(node, scope)) return;
     }
 
+    // Check if it is a static get or set
+    if (node.static && ["get", "set"].indexOf(node.kind) > -1) {
```

Comment:
Can you use `node.static && (node.kind === "get" || node.kind === "set")`? It is just a little longer but it is more clear.